package.json

```
@@ -3,7 +3,7 @@
   "workspaces": [
     "demo/"
   ],
-  "version": "4.1.5",
+  "version": "4.2.0",
```

```suggestion
  "version": "4.1.5",
```

This should be reverted. bumps are automated by my [@favware/cliff-jumper](https://github.com/favware/cliff-jumper) through `yarn bump`